crates/gossip/src/gossip.rs

```diff
@@ -879,6 +879,7 @@ mod test {
     }
 
     #[tokio::test]
+    #[ignore = "This is the wrong place to test this. Check it as a unit test of the gossip message handlers instead. Cannot prevent multiple gossip rounds happening fast any more."]
```

Comment:
Will need to write a new test for this and not make it an integration test where tasks are running